benchmark/benchmain/main.go

```diff
@@ -22,14 +22,20 @@ Package main provides benchmark with setting flags.
 An example to run some benchmarks with profiling enabled:
 
 go run benchmark/benchmain/main.go -benchtime=10s -workloads=all \
-  -compression=on -maxConcurrentCalls=1 -traceMode=false \
+  -compression=on -maxConcurrentCalls=1 -trace=off \
   -reqSizeBytes=1,1048576 -respSizeBytes=1,1048576 \
   -latency=0s -kbps=0 -mtu=0 \
-  -cpuProfile=cpuProf -memProfile=memProf -memProfileRate=10000
+  or -networkMode=Local \
```

Comment:
This is no longer a valid command.

Let's make sure you can copy/paste this into a terminal and it does something useful.

Just delete the latency/kbps/mtu flags here; we can keep those in the binary, but only for unusual use-cases.

Reply:
Done.